src/extension/fake_id0/fake_id0.c

```diff
@@ -1079,6 +1079,10 @@ int fake_id0_callback(Extension *extension, ExtensionEvent event, intptr_t data1
 		Config *config;
 		int uid, gid;
 
+#ifdef USERLAND
```

Comment:
Do we use this definition everywhere we make calls to read_ and write_ meta files now? Otherwise, what happens when access to the hash is attempted?